src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "Tigerdreams4234*");
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Cannot load the JDBC driver", e);
+        }
+    }
+
+    public static Connection getConnection() throws SQLException {
```

Comment:
Don't use schema's name in your queries, because you are configuring it while establishing a connection with DB. Ensure that the `DB_URL` includes the database schema, and it is not hardcoded elsewhere in your SQL queries.